meson.build

```diff
@@ -0,0 +1,27 @@
+project('tinyendian', 'd')
+
+project_version      = '0.1.2'
+project_soversion    = '0'
+
+src_dir = include_directories('source/')
+pkgc = import('pkgconfig')
+
+tinyendian_src = [
+    'source/tinyendian.d'
+]
+install_headers(tinyendian_src, subdir: 'd/')
+
+tinyendian_lib = static_library('tinyendian',
+        [tinyendian_src],
+        include_directories: [src_dir],
+        install: true,
+        version: project_version,
+        soversion: project_soversion,
+        pic: true
```

Comment:
Apparently, Meson might decide to build just with PIE or even disable PIC on some architectures/OSes for static libraries. Later, that is not a problem if you want to link the lib into an executable, but linking it into another shared library (like D-YAML!) fails.
I was also a bit surprised by this behavior, but the pic flag seems to be necessary.